integration-tests/test-app/mirror-input.js

```diff
@@ -0,0 +1,24 @@
+const { registerHtml, useGlobalStore } = require('../../src/tram-one')
```

Comment:
This is more of a personal nit, but I generally try to avoid having these reusable separated test files, as it becomes frustrating when trying to work on additional use cases. You have to check if the test file is used anywhere else, or if it's okay to use our modify it for other tests... I just find it simpler to include it with the test file. But I don't know how other tests here are set up and if that would be an annoying amount of additional code.

Reply:
Oof nvm. If these are integration tests then this is fine. Thought this was unit tests

Reply:
previously I had an app for each test, and that was way more of a nightmare to manage (potentially because I was working on breaking changes, but also because it wasn't clear or obvious which app did what).

Ideally, I'd have a single app that fully tested all of tram-one, that was semantically obvious and meaningful. That's obviously hard to come up with...

Reply:
I'm using a unit test framework to basically write integration tests